samples/create-batch-prediction-job-video-classification.js

```diff
@@ -0,0 +1,255 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function main(
+    batchPredictionDisplayName,
+    modelId,
+    gcsSourceUri,
+    gcsDestinationOutputUriPrefix,
+    project,
+    location = 'us-central1',
+) {
+  // [START aiplatform_create_batch_prediction_job_video_classification]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.\
+   * (Not necessary if passing values as arguments)
+   */
+
+  // const batchPredictionDisplayName = 'YOUR_BATCH_PREDICTION_DISPLAY_NAME';
+  // const modelId = 'YOUR_MODEL_ID';
+  // const gcsSourceUri = 'YOUR_GCS_SOURCE_URI';
+  // const gcsDestinationOutputUriPrefix = 'YOUR_GCS_DEST_OUTPUT_URI_PREFIX';
+  //    eg. "gs://<your-gcs-bucket>/destination_path"
+  // const project = 'YOUR_PROJECT_ID';
+  // const location = 'YOUR_PROJECT_LOCATION';
+
+  // Imports the Google Cloud Job Service Client library
+  const {JobServiceClient} = require('@google-cloud/aiplatform');
+
+  // Specifies the location of the api endpoint
+  const clientOptions = {
+    apiEndpoint: 'us-central1-aiplatform.googleapis.com',
+  };
+
+  // Instantiates a client
+  const jobServiceClient = new JobServiceClient(clientOptions);
+
+  async function createBatchPredictionJobVideoClassification() {
+    // Configure the parent resource
+    const parent = `projects/${project}/locations/${location}`;
+    const modelName = `projects/${project}/locations/${location}/models/${modelId}`;
+    // Values should match the input expected by your model.
+    // confidenceThreshold(float), maxPredictions(number), \
+    // segmentClassification(boolean), shotClassification(boolean) \
+    // oneSecIntervalClassification(boolean)
+    const modelParameters = {
+      structValue: {
+        fields: {
+          confidenceThreshold: {floatValue: 0.5},
+          maxPredictions: {numberValue: 10000},
+          segmentClassification: {boolValue: true},
+          shotClassification: {boolValue: true},
+          oneSecIntervalClassification: {boolValue: true}
+        }
+      }
+    };
+    const inputConfig = {
+      instancesFormat: 'jsonl',
+      gcsSource: {uris: [gcsSourceUri]},
+    };
+    const outputConfig = {
+      predictionsFormat: 'jsonl',
+      gcsDestination: {outputUriPrefix: gcsDestinationOutputUriPrefix},
+    };
+    const batchPredictionJob = {
+      displayName: batchPredictionDisplayName,
+      model: modelName,
+      modelParameters: modelParameters,
+      inputConfig: inputConfig,
+      outputConfig: outputConfig,
+    };
+    const request = {
+      parent,
+      batchPredictionJob,
+    };
+
+    // Create batch prediction job request
+    const [response] = await jobServiceClient.createBatchPredictionJob(request);
+
+    console.log(`Create batch prediction job video classification response`);
+    console.log(`\tName : ${response.name}`);
```

Comment:
So thats a lot of stuff :) Should we consider something like `console.log(util.inspect(response))`?

Reply:
This decision preceded me, but I think the reasoning was that developers often don't really run the samples so much as read them looking for a significant property here and there.